Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -251,4 +251,53 @@ - (void)handleRemoteNotificationsRegistered:(NSNotification *)notification
   }
 }
 
+RCT_EXPORT_METHOD(getScheduledLocalNotifications:(RCTResponseSenderBlock)callback)
+{
+  NSArray<UILocalNotification *> *scheduledLocalNotifications = [UIApplication sharedApplication].scheduledLocalNotifications;
+
+
+  NSMutableArray *formattedScheduledLocalNotifications = [[NSMutableArray alloc] init];
+
+  for (UILocalNotification *notification in scheduledLocalNotifications) {
+
+    NSMutableDictionary* formattedScheduledLocalNotification = [NSMutableDictionary dictionary];
+
+    if (notification.fireDate) {
+      NSDateFormatter *formatter = [[NSDateFormatter alloc] init];
+      [formatter setDateFormat:@"yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ"];
+      NSString *fireDateString = [formatter stringFromDate:notification.fireDate];
+
+      [formattedScheduledLocalNotification setObject: fireDateString forKey:@"fireDate"];
```

Comment:
Can you use modern Obj-C here? `formattedScheduledLocalNotification[@"fireData"] = fireDateString;`